docs/onnx_custom_op/main.cpp

```diff
@@ -0,0 +1,160 @@
+#include <cstring>
```

Comment:
This code looks like a test. Maybe we can split it on two parts: library which is used for documentation and new test which should check this code.

Reply:
I'd prefer not to split it. I wanted to create a sample executable that has all the pieces to execute an inference for a simple model.

Reply:
Ok, it is good point.

How we can test this executable that it works well?

Reply:
I plan to add some internal tests. But that can happen after this PR is merged

Reply:
In this case we need to have a ticket in order to don't forget about it, and also I have a question about assert in the main function.